tests/virt_autotest/virt_autotest_base.pm

```diff
@@ -95,7 +95,7 @@ sub execute_script_run {
 
     if ($ret) {
         save_screenshot;
-        $ret =~ s/$pattern//g;
+        $ret =~ s/[\r\n]+$pattern[\r\n]+//g;
```

Comment:
This change just clean up the invisible character cause by verify code.(echo CMD-FINISHED-xxxxx) , and this PR require this to get output inside the worker.
I'm ok that cancel this change , and open another pr before this pr